storages/backends/azure_storage.py

```diff
@@ -57,9 +57,20 @@ def size(self, name):
         return properties["content-length"]
 
     def _save(self, name, content):
+        if hasattr(content.file, 'content_type'):
```

Comment:
This assumes we have a `ContentFile` right? I know that's what we return from `_open` but is this going to be the case generally?